CHANGELOG.md

```diff
@@ -18,6 +18,16 @@ end
 
 Until now it was possible to see if an email account was registered in Decidim, by using features like "Forgot your password", as the response changed if the email existed ("`You will receive an email with instructions on how to reset your password in a few minutes`") that's different to a non-existing user account ("`could not be found. Did you sign up previously?`"). This allows User Enumration attacks, where a malicious actor can check if anyone has an acount in the platform. As per [\#8537](https://github.com/decidim/decidim/pull/8537), anyone has the same answer always "`If your email address exists in our database, you will receive a password recovery link at your email address in a few minutes`".
 
+#### Global search user by nickname
```

Comment:
You'll need to resolve conflicts with CHANGELOG, as yesterday I started with the preparation for the v0.26.0 release